.eslintignore

```diff
@@ -0,0 +1,10 @@
+node_modules
+.loom
+/polaris-react/build
+/polaris-react/build-internal
+/polaris.shopify.com/.next
+
+
+# These folders should pass linting, remove these lines and fix the errors
```

Comment:
Task for a follow-up PR: Remove these lines and opt these folders into being linted